### DISCUSSION TITLE
Hard (sticky) to switch layouts.

### DISCUSSION THREAD

**Discussion by tomatojam42** [valinet/ExplorerPatcher#1627] [Q&A]
Hello.
Thanks for the great software, it's really useful when I can see name of each window and can aims to it by mouse.
But there is annoying little bug that corrupts my experience.

When I push Alt+Shift to switch layout, it may not be switched. Just indicator blink shortly like it **tried** but something went wrong. Like Patcher unable to process key combination in time.

I had try different ways to push buttons: [Alt] first, [Shift] first, same time, shortly, long press — anyway I have to try 2...5 times before language switches. In case of technical writer it's failure.

Also I have 3 layouts: ENG US, ENG GE, RU

When I delete ExplorerPatcher the bug came away.

Please fix that. Thanks!

**Comment by pyrates999** [comment]
Please post windows version and build.
Please post explorer patcher version.

**Reply by tomatojam42** [reply]
Windows 11 22H2 (Home), build 22621.1848
ExplorerPatcher latest [22621.1555.55.1](https://github.com/valinet/ExplorerPatcher/releases/tag/22621.1555.55.1_a95a688)

3 layouts because German labels of keyboard: ENG US, ENG GE, RU

**Reply by pyrates999** [reply]
There's a windows update available for today, July 11th.  Perhaps that will fix this?

**Reply by tomatojam42** [reply]
I've update Windows just now. Now it's 22621.1992. Then installed ExplorerPatcher and no positive effect - bug appears again.
Yes, I understand that you have to verify all stupid situations (when trouble evaporates by itself). Any more?..

Upd: can someone move this topic in right branch?

**Reply by pyrates999** [reply]
Try the following:

1. uninstall explorer patcher
2. delete the registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
3. reboot
4. install explorer patcher
5. verify you get a notification that symbols have been downloaded
6. restart windows explorer using either the task manager or in the properties window of explorer patcher
7. reboot

Then apply your settings again in explorer patcher to see if it fixes it.
